core/sql/optimizer/mdam.cpp

```diff
@@ -2007,39 +2007,39 @@ void Disjuncts::computeCommonPredicates()
       get(disjunct,i);
       if(CmpCommon::getDefault(RANGESPEC_TRANSFORMATION) == DF_ON )
       {
-	    ValueIdSet inVidset = disjunct.getAsValueIdSet();
-	    ValueIdSet outVidset,parsedVs;
-        for (ValueId predId = inVidset.init();
-	     inVidset.next(predId);
-	     inVidset.advance(predId) )
-	    {
-         if(predId.getItemExpr()->getOperatorType() == ITM_RANGE_SPEC_FUNC )
-         {
-		  if(predId.getItemExpr()->child(1)->getOperatorType() == ITM_AND ){
-	        predId.getItemExpr()->child(1)->convertToValueIdSet(parsedVs, NULL, ITM_AND, FALSE);
+	 ValueIdSet inVidset = disjunct.getAsValueIdSet();
+	 ValueIdSet outVidset,parsedVs;
+         for (ValueId predId = inVidset.init();
+	      inVidset.next(predId);
+	      inVidset.advance(predId) )
+	 {
+            if(predId.getItemExpr()->getOperatorType() == ITM_RANGE_SPEC_FUNC )
+            {
+	       if(predId.getItemExpr()->child(1)->getOperatorType() == ITM_AND ){
+	          predId.getItemExpr()->child(1)->convertToValueIdSet(parsedVs, NULL, ITM_AND, FALSE);
 		    outVidset +=parsedVs;
-		 }
-	     else if(predId.getItemExpr()->child(1)->getOperatorType() != ITM_AND 
+            }
+	    else if(predId.getItemExpr()->child(1)->getOperatorType() != ITM_AND 
 			 && predId.getItemExpr()->child(1)->getOperatorType() != ITM_OR)
-	      outVidset += predId.getItemExpr()->child(1)->castToItemExpr()->getValueId();	    
-	   }
+	       outVidset += predId.getItemExpr()->child(1)->castToItemExpr()->getValueId();	    
+           }
 	   else
-	    outVidset +=predId;
-	    parsedVs.clear();
+	     outVidset +=predId;
+	 parsedVs.clear();
 	}
 
 	if (i==0)
 	  commonPredicates_.insert(outVidset);
 	else
 	  commonPredicates_.intersectSet(outVidset);
-      }
-      else
-      {
+     }
+     else
+     {
 	if (i==0)
 	  commonPredicates_.insert(disjunct.getAsValueIdSet());
 	else
 	  commonPredicates_.intersectSet(disjunct.getAsValueIdSet());
-      }
+     }
```

Comment:
Yes. there is no real change in this file. I aligned the blocks.

On Thu, Aug 27, 2015 at 11:00 AM, Suresh Subbiah <notifications@github.com>
wrote:

> In core/sql/optimizer/mdam.cpp
> <https://github.com/apache/incubator-trafodion/pull/66#discussion_r38112704>
> :
>
> >  	if (i==0)
> >  	  commonPredicates_.insert(disjunct.getAsValueIdSet());
> >  	else
> >  	  commonPredicates_.intersectSet(disjunct.getAsValueIdSet());
> > -      }
> > +     }
>
> just for my understanding, there is no change in this file and we the
> diffs are all whitespaces. I am sorry I do not know how to ignore
> whitespace in this diff tool.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/66/files#r38112704>.
>



-- 
Regards, --Qifan